docs/getting-started.md

```diff
@@ -25,8 +25,8 @@ The generic requirements are explained below:
 
 - **Linux 4.14+**
 
-  Firecracker currently supports physical Linux x86_64 hosts, running kernel
-  version 4.14 or later.
+  Firecracker currently supports physical Linux **x86_64** and **aarch64**
```

Comment:
We should specify that aarch64 is not feature complete at it is considered alpha.

Reply:
Done, take a look.